### ISSUE TITLE
question to training

### ISSUE THREAD

**Issue by Arnold1** [tensorflow/decision-forests#163]
Hi, i have a small question:

- distributed training:
https://github.com/tensorflow/decision-forests/blob/main/documentation/distributed_training.md
I saw Gradient Boosted Trees are supported for sure - but what about tfdf.keras.RandomForestModel, etc.?
